### PR TITLE
Improvements to 1.3 section & document structure in general & add some of the regular-issue SUSE boilerplate

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -5,9 +5,10 @@ This repository contains additional notes for the releases of SUSE Cloud Applica
 * link:/adoc/Release-Notes.adoc[Release Notes] for specific releases
 * link:/adoc/known-issues.adoc[Known issues] for known issues
 
-Some hints for special environments:
+Hints for specialized environments:
 
-* link:/adoc/Notes-CaaSP.adoc[SUSE CaaS (Container as a Service) Platform]
+* link:https://www.suse.com/documentation/suse-caasp-2/book_caasp_deployment/data/book_caasp_deployment.html[SUSE CaaS (Container as a Service) Platform & Bare Metal or Non-OpenStack Virtualization Platforms]
```
Comment (btat) [inline README.adoc]: This should point to CaaSP 3

File: README.adoc [hunk 2]
```diff
@@ -5,9 +5,10 @@ This repository contains additional notes for the releases of SUSE Cloud Applica
...
+* link:https://www.suse.com/documentation/suse-openstack-cloud-8/book_install/data/sec_heat_templates_install.html[SUSE CaaS (Container as a Service) Platform & SUSE OpenStack Cloud]
```
Comment (btat) [inline README.adoc]: Getting a 404 for this link

File: README.adoc (hunk 1 above)
Comment (ghost) [reply README.adoc]: Thanks! Fixed.

File: README.adoc (hunk 2 above)
Comment (ghost) [reply README.adoc]: Thanks! The book has been renamed from book_install -> book_installation at some point.

**Review by gaktive — APPROVED** [review]

**Comment by gaktive** [comment]
LGTM

**Pull request merged**